07/ID_tryExcept

```diff
@@ -0,0 +1,53 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+
+# MONTHS 30 DAYS
+months_30 = [4, 6, 9, 11]
+
+# MONTHS 31 DAYS
+months_31 = [1, 3, 5, 7, 8, 10, 12]
+
+# YEARS
+years = list(range(86, 100)) + list(range(00, 23))
+ID_only_numbers = []
+
+while True:
+
+    ID = input('Napiš rodné číslo: ')
+    if ID[6] == "/":
+        try:
+            a = int(ID[:6])
+            b = int(ID[7:])
+        except ValueError:
```

Comment:
ValueError ti program vráti ak tam nebudeš mať čísla ale iné znaky, takže hláška by mala byť trochu iná. Zároveň ak chceš skontrolovať že tam zadal správny počet znakov (t.j. 11) tak by si mala overovať celkovú dĺžku (11) a na r. 21 by si mala chcieť 4 znaky, pretož takto sa môže stať že ti prejde aj číslo 123456/7 a spadne to až úplne na konci.